docs/intro-vs-pony.md

```diff
@@ -27,6 +27,14 @@ A declaration "head" is any such line beginning with a colon-prefixed identifier
   :fun greeting: "Hello, World!"
 ```
 
+### Functions
+
+Unlike functions in pony, in Mare:
+* function declaration doesn't require parenthesis if there are no arguments
+* function call doesn't require parenthesis if there are no arguments
+* it's mandatory to put a whitespace between function name and it's parameters
```

Comment:
I intend to probably remove this restriction later, but yeah, it's good to document it here for now.